src/pages/MessageReviewPage.tsx

```
@@ -1,15 +1,90 @@
-import React from "react";
+import { useLazyQuery, useQuery } from "@apollo/client";
+import ms from "date-fns/esm/locale/ms/index.js";
+import { uniqBy } from "lodash";
+import React, { useEffect, useState } from "react";
+import { Connection } from "src/api/graphql";
 import Header from "src/components/Header";
+import Timeline from "src/components/Timeline";
+import { TimelineEvent, TimelineMessageEvent } from "src/components/Timeline/TimelineContent";
+import { TConnectionSidebarItem } from "src/components/Timeline/TimelineSidebar";
+import { GetConnectionDocument } from "src/graphql/GetConnection.generated";
+import { GetPendingMessagesDocument } from "src/graphql/GetPendingMessages.generated";
+import { useAppSelector } from "src/redux";
+import { genFullName } from "src/utils";
 
 interface Props {}
 
 const MessageReviewPage = (props: Props) => {
+  const facilityId = useAppSelector((state) => state.facilities.selected?.id);
+
+  const { error: pendingMessageError, data: pendingMessageData } = useQuery(
+    GetPendingMessagesDocument,
+    {
+      variables: { id: facilityId || "" },
```

we should eliminate the null case of `facilityId` some other way. maybe throw an error? this would cause an error anyway, but a less helpful one.